packages/react/src/alert-dialog/portal/AlertDialogPortal.tsx

```diff
@@ -0,0 +1,75 @@
+'use client';
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { FloatingPortal } from '@floating-ui/react';
+import { useAlertDialogRootContext } from '../root/AlertDialogRootContext';
+import { refType, HTMLElementType } from '../../utils/proptypes';
+
+/**
+ *
+ * Demos:
+ *
+ * - [Alert Dialog](https://base-ui.com/components/react-alert-dialog/)
+ *
+ * API:
+ *
+ * - [AlertDialogPortal API](https://base-ui.com/components/react-alert-dialog/#api-reference-AlertDialogPortal)
+ */
+const AlertDialogPortal: AlertDialogPortal = function AlertDialogPortal(
+  props: AlertDialogPortal.Props,
+) {
+  const { children, container, keepMounted = false } = props;
+
+  const { mounted } = useAlertDialogRootContext();
```

Comment:
We agreed to use the term `visible` instead of `mounted`.

Reply:
`mounted` is used everywhere currently and doesn't relate to this PR. We can change it but it's not necessary/not exposed to publicly right now